client/src/actions/index.js

```diff
@@ -41,8 +44,18 @@ export const pageCounter = (counter, direction) => {
   }
 
   return { type: PAGE_COUNTER, payload: count };
-}
+};
 
 export const recordHistory = history => {
   return { type: CHECK_HISTORY, payload: history };
-}
+};
+
+export const emailSending = email => {
+  axios
+    .post('/api/send_email', email)
+    .then(res => console.log(res))
```

Comment:
Instead of a console log here, shouldn't that be the time where the report is sent to the database to be saved?

Reply:
Hoslack already fixed that. Check his commits under "Save Reports" branch